### PR TITLE
f-content-cards@7.4.0 - f-braze-adapter update

### PR DESCRIPTION
**Pull request by dandel10n**
### Changed
- Updated f-braze-adapter package version to support new `$log` interface.
- Reame update.

### Removed
- `badge()` styles import as outdated and unused.

### CODE REVIEW

File: packages/components/organisms/f-content-cards/README.md [hunk 1]
```diff
@@ -198,17 +198,14 @@ Storybook can be used to develop new and existing components.
 
 To start storybook:
 
-> Please ensure you are in the f-content-card directory as outlined in the above instructions.
+> Please ensure you are in the root level of the momorepo.
```
Comment (dandel10n) [inline packages/components/organisms/f-content-cards/README.md]: Oh, I will fix a typo in a minute

File: packages/components/organisms/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue [hunk 2]
```diff
@@ -171,8 +170,6 @@ export default {
 </script>
 
 <style lang="scss" module>
-    @include badge();
```
Comment (matthewhardern) [inline packages/components/organisms/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue:174]: what does this remove?

Comment (dandel10n) [reply packages/components/organisms/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue:174]: I couldn't find the use of this badge styles and they've been deprecated in v6 of fozzie. When running locally there is a warning about it, so I removed it as unused and outdated. Let me know if I missed something and these styles are still needed.

Comment (dandel10n) [reply packages/components/organisms/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue:174]: At the moment it doesn't add anything, only a warning for the build that this mixin is deprecated from [here](https://github.com/justeat/fozzie/blob/master/src/scss/components/optional/_badges.scss)

Comment (matthewhardern) [reply packages/components/organisms/f-content-cards/src/components/cardTemplates/voucherCard/VoucherCard.vue:174]: cool sounds good to me then.

**Review by matthewhardern — DISMISSED** [review]

**Review by matthewhardern — APPROVED** [review]

**Review by AlixRichards — APPROVED** [review]

**Pull request merged**